README.md

```diff
@@ -14,6 +14,82 @@ This package defines some tests which invoke `perf_test` from Apex.AI's [perform
   - rmw_fastrtps_dynamic_cpp
   - rmw_opensplice_cpp
 
+### Test 1 - Performance Test  (APEX.AI)
+
+In this test we are testing the Performance Test provided by APEX.AI. Right now we have [our own fork](https://github.com/ros2/performance_test) because there are some pending pull requests in the official gitlab repository.
```

Comment:
We're not really testing it, we're using it.

```suggestion
In this test we are running the Performance Test provided by APEX.AI. Right now we have [our own fork](https://github.com/ros2/performance_test) because there are some pending pull requests in the official gitlab repository.
```